client/router_create.go

```diff
@@ -420,11 +433,18 @@ func ClusterRoles() []*rbacv1.ClusterRole {
 			{
 				APIGroups: []string{"skupper.io"},
 				Resources: []string{"skupperclusterpolicies"},
-				Verbs:     []string{"get", "list", "watch"}},
+				Verbs:     []string{"get", "list", "watch"},
+			},
 			{
 				APIGroups: []string{""},
 				Resources: []string{"namespaces"},
-				Verbs:     []string{"get"}},
+				Verbs:     []string{"get"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"nodes"},
+				Verbs:     []string{"get", "list", "watch"},
```

Comment:
Update will log a message rather than fail. Generally, will need to provide documentation on how to determine if role is properly enabled for monitoring nodes.